### DISCUSSION TITLE
Migration of Framework: Protractor to Webdriverio

### DISCUSSION THREAD

**Discussion by Subhanak991** [webdriverio/codemod#78] [Migration Issue]
After Migrating one of the test file from protractor, the find elements gets changed to $ and also some other changes happens. But when I try to execute the Test Cases it's showing "Error:  $ is not defined". Since before the locators are not written in get form and initialization was completely different. Please suggest how should I proceed in such situation.

**Comment by christian-bromann** [comment]
> Please suggest how should I proceed in such situation.

It is recommended to move element fetch calls into getters.

**Reply by Subhanak991** [reply]
Thanks for the reply
